librz/core/cconfig.c

```diff
@@ -3340,7 +3340,7 @@ RZ_API int rz_core_config_init(RzCore *core) {
 
 	SETCB("dbg.libs", "", &cb_dbg_libs, "If set stop when loading matching libname");
 	SETBPREF("dbg.skipover", "false", "Make dso perform a dss (same goes for esil and visual/graph");
-	SETI("dbg.hwbp", 0, "Set HW or SW breakpoints");
+	SETB("dbg.hwbp", false, "Set HW or SW breakpoints");
```

Comment:
Maybe "use" instead of "set"?

Reply:
What about "Always use hardware instead of software breakpoints"? I am planning to automatically fallback to hardware breakpoints if software fails (bc not possible on a certain arch for example) on `dbg.hwbp=false` and then this description would not contradict that.

Reply:
Sounds good too.